controller/service.rst

```diff
@@ -83,6 +81,74 @@ Then you can define it as a service as follows:
 
         $container->register('app.hello_controller', HelloController::class);
 
+Using the controller shortcuts
+------------------------------
+
+To use the traditional controller shortcuts, you can use the
```

Comment:
this paragraph is a bit strange to me, it does really teach enough to the reader imho.
eg
> if you prefer explicit injection (you can't access the container)

"why would I prefer explicit injection? esp when the words in the brackets suggest that this is less powerful, so let's just to the previous `Controller`." Which then raises "why did they tell me about this? do I miss anything?" leading to confusion.

my 2cts :)